eth-bytecode-db/eth-bytecode-db/migration/src/lib.rs

```diff
@@ -12,6 +13,7 @@ impl MigratorTrait for Migrator {
         vec![
             Box::new(m20220101_000001_initial::Migration),
```

Comment:
by the way, do we need separate migrations even before the first release? may be merge them into one migration? just a thought

Reply:
1. It was convenient for me to UP and DOWN migration with indexes to test my big SQL with different indexes.

2. I understand your concern about big amount of migrations, but I think if migration was commited in main, then someone (staging server?) could pull this branch and apply migrations. In this case we shouldn't rewrite this migration

Reply:
I agree with @leviska. I believe we may change it as we want before the first release. There is no eth-bytecode-db service on the staging server right now and merging all migrations into the first one seems to be a good idea for me. 

It would be better to do that in a separate PR, though, when verification code would be merged as well. There are some more migrations in verification code, so probably, it would be better to merge them together

Reply:
yeah, let's just merge them may be before release